CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Return global config ready status to downstream relays. ([#2765](https://github.com/getsentry/relay/pull/2765))
 - Add Mixed JS/Android Profiles events processing. ([#2706](https://github.com/getsentry/relay/pull/2706))
 - Allow to ingest measurements on a span. ([#2792](https://github.com/getsentry/relay/pull/2792))
+- Org rate limit metrics per batch. ([#2758](https://github.com/getsentry/relay/pull/2758))
```

Comment:
- Let's update the entry to reflect that this is actually rate limiting per bucket
- I would propose moving this to "Internal"